dotCMS/src/main/java/com/dotcms/rest/api/v1/contenttype/FieldResource.java

```diff
@@ -66,6 +62,45 @@ public Response createContentTypeField(@PathParam("typeId") final String typeId,
 		Response response = null;
 		
 		try {
+			final List<Field> fields = new JsonFieldTransformer(fieldsJson).asList();
+
+			for (Field field : fields) {
+				field = fapi.save(field, user);
+			}
+
+			List<Field> contentTypeFields = fapi.byContentTypeId(typeId);
+			response = Response.ok(new ResponseEntityView(new JsonFieldTransformer(contentTypeFields).mapList())).build();
+		} catch (DotStateException e) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Either log or rethrow this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1166)